### PR TITLE
ft(get Message): user can get a message

### PR DESCRIPTION
**Pull request by beejay1293**
### What does this PR do?

- get specific message

### Description of the task to be completed?

- set up getMessage route and controller
- Write the test for getMessage
- Document on swagger.yaml

### Any background context you want to provide?

N/A

### What are the relevant Github issues?

N/A

### Questions:

N/A

### CODE REVIEW

File: server/controller/message/messages.js [hunk 1]
```diff
@@ -105,6 +105,59 @@ class MessageController {
...
+
+      const sentMessage = await Message.findOne({
+        where: { senderId: userId, id: messageId }
+      });
+
+      const receivedMessage = await Message.findOne({
+        where: { receiverId: userId, id: messageId }
+      });
+
+      if (!sentMessage && !receivedMessage) {
```
Comment (fortunee) [inline server/controller/message/messages.js:138]: What if there's a  `sentMessage` and no `receivedMessage`? 🤔 @beejay1293

**Review by fortunee — COMMENTED** [review]
Nice work! Bolaji. I left one comment for you

File: server/controller/message/messages.js (hunk 1 above)
Comment (beejay1293) [reply server/controller/message/messages.js:138]: @fortune, that was why I checked in the next line with this:
  if (!sentMessage && !receivedMessage) {
        return res.status(404).json({
          status: 404,
          message: 'no messages'
        });
      }
so, if there's a SentMessage and no receivedMessage, the condition of the above code would be false, and the next line would run instead.


and code below helps me to check which one contains a message to return to the user
 
if (sentMessage) {
        message = sentMessage.dataValues;
      } else {
        message = receivedMessage.dataValues;
      }

Comment (fortunee) [reply server/controller/message/messages.js:138]: Alright please resolve the conflicts @beejay1293

Comment (beejay1293) [reply server/controller/message/messages.js:138]: Done @fortunee

**Review by fortunee — APPROVED** [review]
LGTM

**Pull request merged**